src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -102,8 +104,23 @@ export class Accept extends Component<AcceptProps, AcceptState> {
         case "capture_failed": {
           this.onMutationError(
             new ErrorWithMetadata(error.code, error),
-            "An error occurred",
-            "There was an error processing your payment. Please try again or contact orders@artsy.net."
+            "Charge failed",
+            "Payment authorization has been declined. Please contact your card provider and try again.",
```

Comment:
Yeah, re: your comment on the Jira ticket. Let's remove this case for now since we're still trying to decide how to message + when to route them to the new payment page.

I think you can keep the messaging below on the `NewPayment` page!

Reply:
ok, done!